.github/workflows/README.md

```diff
@@ -114,6 +115,82 @@ jobs:
       github-token: "${{ secrets.GH_TOKEN }}"
 ```
 
+## Kustomize Release
+
+This workflow will deploy to GKE using a Kustomize root directory.
+
+### Prerequisites
+
+Set up GitHub pages for your repository in Settings → Pages → Build and deployment source → GitHub Actions. A special `gh-pages` branch is not needed, since we will use GitHub actions to deploy a Pages artifact.
+
+Currently it is not possible to download a previously created Pages artifact as they quickly expire after deploying it. When releasing an update to a Helm chart, we want to keep all previous versions of the Helm chart available. Therefore, as a workaround, we download the index.yaml file from Pages, parse all referenced releases, and download these .tgz packages from Pages as well. Then we package the new version and update the index. Afterwards, a new Pages artifact is created from these files and finally deployed.
+
+### Dependencies
+
+This workflow is built from multiple composite actions listed below:
+
+- [helm-lint](https://github.com/bakdata/ci-templates/tree/main/actions/helm-lint)
+- [bump-version](https://github.com/bakdata/ci-templates/tree/main/actions/bump-version)
+- [helm-package](https://github.com/bakdata/ci-templates/tree/main/actions/helm-package)
+- [commit-and-push](https://github.com/bakdata/ci-templates/tree/main/actions/commit-and-push)
+
+### Input Parameters
+
+| Name             | Required |             Default Value             |  Type  | Description                                                                                                                                |
+| ---------------- | :------: | :-----------------------------------: | :----: | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| kustomization-path    |    ✅    |       -       | string | Path to the root directory of the kustomization                                                           |
+| timeout  |    ❌    |       60       | string | Time out(in seconds) for custom ressource definitions                                                            |
+| python-version              |    ❌    | "3.10" | string | The required python version                                                                                                    |
+| gcloud-sdk-version |    ❌    |      "376.0.0"       | string | The required gcloud-sdk version |
+| kubectl-version     |    ❌    |               "v1.23.0"               | string | The required kubectl version                                                                                                                           |
```

Comment:
Please remove the `required` part from the description since the versions are not required to set for the workflow